pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -41,8 +51,30 @@ func (csh *cisScanHandler) Create(cs *v3.ClusterScan) (runtime.Object, error) {
 	}
 	if !v3.ClusterScanConditionCreated.IsTrue(cs) {
 		logrus.Infof("cisScanHandler: Create: deploying helm chart")
+
+		appInfo := &appInfo{
+			appName:     cs.Name,
+			clusterName: cs.Spec.ClusterID,
+		}
+		if cs.Spec.ScanConfig.CisScanConfig != nil {
+			appInfo.skip = strings.Join(cs.Spec.ScanConfig.CisScanConfig.Skip, ",")
+			if cs.Spec.ScanConfig.CisScanConfig.DebugMaster {
+				appInfo.debugMaster = "true"
+			}
+			if cs.Spec.ScanConfig.CisScanConfig.DebugWorker {
+				appInfo.debugWorker = "true"
+			}
+		}
+		// Check if the configmap is populated
+		cm, err := csh.userCtxCMLister.Get(v3.DefaultNamespaceForCis, v3.ConfigMapNameForUserConfig)
+		if err != nil && !kerrors.IsNotFound(err) {
+			return cs, fmt.Errorf("error fetching cm: %v", err)
```

Comment:
can we say config map? :)

Reply:
`kubectl get cm` :) it's a well known acronym ... ha ha

Reply:
anyways, will change it :D

Reply:
on a second thought, we can just return the raw api error as it will be self explanatory

Reply:
Used full word and matched the format with the rest of the function.